src/BannedSymbols.txt

```diff
@@ -9,3 +9,5 @@ M:Microsoft.VisualStudio.ProjectSystem.CommonProjectSystemTools.LoadedProject(Mi
 
 T:Microsoft.VisualStudio.ProjectSystem.ProjectAutoLoadAttribute; Using IProjectDynamicLoadComponent avoids accidently reloading the project while loading, switching branches, installing NuGet package, etc.
 T:Microsoft.VisualStudio.ProjectSystem.ConfiguredProjectAutoLoadAttribute; Using IProjectDynamicLoadComponent avoids accidently reloading the project while loading, switching branches, installing NuGet package, etc.
+
+M:System.Threading.Tasks.Task.ConfigureAwait(System.Boolean); "ConfigureAwait(true)" should be removed, and "ConfigureAwait(false)" should be replaced with "await TashScheduler.Default"
```

Comment:
❔  Should this also include `Task<TResult>.ConfigureAwait`?

Reply:
I believe so. Even though it's a subclass, it returns `Task<T>` so must be hiding the base.

Reply:
This highlighted a bug in the analyzer around generic types/methods. Fixed in dotnet/roslyn-analyzers#1927.